Policies/github-terms-of-service.md

```diff
@@ -126,7 +126,7 @@ You retain all moral rights to Your Content that you upload, publish, or submit
 To the extent this agreement is not enforceable by applicable law, you grant GitHub the rights we need to use Your Content without attribution and to make reasonable adaptations of Your Content as necessary to render the Website and provide the Service.
 
 ### E. Private Repositories
-**Short version:** *You may have access to private repositories. We treat the content of private repositories as confidential, and we only access it for support reasons, with your consent, or if required to for security reasons.*
+**Short version:** *We treat the content of private repositories as confidential, and we only access it as described in our Privacy Statement—for security purposes, to assist the repository owner with a support matter, to maintain the integrity of the Service, to comply with our legal obligations, if we have reason to believe the contents are in violation of the law, or with your consent.*
```

Comment:
>if we have reason to believe the contents are in violation of the law

You always have a reason to beleive that all content of each repo of each user is in violation of the law, just because prior probability of that is greater than zero. So the phrase probably should be replacrd with

>or when our employees just want to look into them